src/main/java/com/manardenza/console/ContentsUserMenu.java

```diff
@@ -0,0 +1,265 @@
+        String cityName = null;
+        try {
+            cityName = String.valueOf(inputDataFromUser("Enter city name: ", false,
+                    false, true, false));
+        } catch (IOException e) {
+            outputDataInConsole("Entered incorrect name of the city");
+        }
+        outputDataInConsole("List of hotels in the city found : " + cityName);
+        List<Hotel> hotelList = hotelController.findHotelByCity(cityName);
+        hotelList.forEach(item -> outputDataInConsole(item.toString()));
+        return hotelList;
+    }
+
+    private Map<String, List<Room>> findRoomMenu() throws IOException {
+        String city = null;
+        Integer person = null;
+        Integer price = null;
+        Date reservedFrom = null;
+        Date reservedTo = null;
+        try {
+            city = String.valueOf(inputDataFromUser("Enter city name: ", false,
+                    false, true, false));
+            person = (Integer) inputDataFromUser("Enter the number of persons: ", true,
+                    false, false, false);
+            price = (Integer) inputDataFromUser("Enter maximum price rate: ", true,
+                    false, false, false);
+            reservedFrom = (Date) (inputDataFromUser("Enter arrival date in the format dd.mm.yyyy: ",
+                    false, false, false, true));
+            reservedTo = (Date) (inputDataFromUser("Enter the departure date in the format dd.mm.yyyy: ",
+                    false, false, false, true));
+            compareDate(reservedFrom, reservedTo);
+        } catch (IOException e) {
+            outputDataInConsole("Enter arrival date in the format");
+        }
+        Map<String, List<Room>> foundRooms = hotelController.getAvailableRooms(city, person, price,
+                reservedFrom, reservedTo);
+        if (foundRooms.values().isEmpty()) {
+            outputDataInConsole("Room with such parameters not found");
+        } else {
+            outputDataInConsole("The list of found rooms:\n ");
+            outputListInConsole(foundRooms.entrySet());
+        }
+        outputDataInConsole("For room reservation please fill out the following form");
+
+        bookRoomMenu();
+        outputSplitLine();
+        return foundRooms; 
+    }
+
+    private Long bookRoomMenu() throws IOException {
+        Long reserveId = 0L;
+        boolean bookFlaf = true;
+        while (bookFlaf) {
+            outputDataInConsole("Do you have booking room?");
+            showUserMenu("Booking menu");
+            Integer bookRoom = (Integer) inputDataFromUser("Enter number: ", true,
+                    false, false, false);
+            Integer numberRoom = null;
+            switch (bookRoom) {
+                case (1):
+                    try {
+                        numberRoom = (Integer) inputDataFromUser("Enter the room number: ", true,
+                                false, false, false);
+                    } catch (IOException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1166)